posts/taking-learning-outside-the-classroom-into-the-cloud.md

```diff
@@ -0,0 +1,54 @@
+---
+authors:
+- Emily Slaughter
+tags:
+- Culture
+- Internship
+- Student
+- LifeAtIppon
+date: 20121-04-29T17:00:00.000Z
+title: "Taking Learning Outside of the Classroom and Into the Cloud"
+image: 
+---
+
+A love for learning is a quality that I have always looked for in peers and teammates, and it’s one that I’ve had the opportunity and privilege to invest in daily over the past 9 months. To me, a commitment to learning requires humility to accept that we often have far more questions than answers to a problem. But when we commit ourselves to learning, we are promising to always seek self-improvement: to honor our questions, seek better understanding, and ultimately be a better version of ourselves today than we were yesterday. In the workplace, a love for learning means we are honest and accountable with our teammates, and more importantly we support and believe in each other’s growth. It was this exact quality that drew me to working with Ippon Technologies, USA.
```

Comment:
In the workplace, a love for learning means we are honest and accountable with our teammates, and more importantly, we support and believe in each other’s growth. It was this exact quality that drew me to working with Ippon Technologies USA.